ScreenControl_Unity/Assets/ScreenControl/Client/Scripts/Connection/ServiceConnection.cs

```diff
@@ -146,38 +146,26 @@ internal void SendMessage(string _message, string _requestID, Action<WebSocketRe
             webSocket.Send(_message);
         }
 
-        // Function: SendMessage
-        // Used internally to send or request information from the Service via the <webSocket>. To
-        // be given a pre-made _message and _requestID. Provides an asynchronous <ConfigStateResponse>
-        // via the _callback parameter.
-        internal void SendMessage(string _message, string _requestID, Action<ConfigStateResponse> _callback)
```

Comment:
Have we fully lost SendMessage now, in favour of RequestConfigState and SendChangeRequest?

Reply:
I thought I did this and I couldn't find where! Will fix!